pkg/resource/notebook_instance/hooks.go

```diff
@@ -48,3 +52,33 @@ func (rm *resourceManager) customSetOutput(r *resource) {
 	latestStatus := r.ko.Status.NotebookInstanceStatus
 	svccommon.SetSyncedCondition(r, latestStatus, &resourceName, &modifyingStatuses)
 }
+
+// CustomSetOutputDescribe has two functions
+// 1. Setting the resource synced condition during sdk.Find
+// 2. Starting the Notebook if the StoppedByController field is set to UpdateTriggerd
+// 3. Reseting the StoppedByController field.
+func (rm *resourceManager) customSetOutputDescribe(r *resource,
+	ko *svcapitypes.NotebookInstance) error {
+	notebookStatus := *ko.Status.NotebookInstanceStatus // Get the Notebook State
+	if notebookStatus == svcsdk.NotebookInstanceStatusStopped {
+		if ko.Status.StoppedByControllerMETA != nil && *ko.Status.StoppedByControllerMETA == "UpdateTriggered" {
```

Comment:
if I remember correctly we decided on the name `StoppedByControllerMetadata`, @surajkota ?

Reply:
I updated it to StoppedByControllerMetadata